docs/developer/extending/apps/synchronous-webhooks.mdx

```diff
@@ -164,6 +164,7 @@ Below is the list of fields that webhook can return in the JSON response for a p
 
 - `action_required` - (boolean) determines if additional action (confirmation) is required.
 - `action_required_data` - (object) additional data that may be needed to confirm the payment, for example, URL to redirect the user to the confirmation page in the frontend.
+- `kind` - (string) Transaction kind, one of: action_to_confirm, auth, cancel, capture, capture_failed, confirm, external, pending, refund, refund_failed, refund_ongoing, refund_reversed, void.
```

Comment:
Hey @JannikZed,
Thanks for pointing out this!
Can you also update below example to include `kind` field? 🙏 
https://github.com/saleor/saleor-docs/pull/467/files#diff-0358f49f3174a451a396b8a613ae5382495a6bb57b9c7a8b9d6b8af999a9f425R184

Reply:
@korycins I added a new example on how to use the "kind" field.

Reply:
Thanks!